spec.txt

```diff
@@ -9757,9 +9757,19 @@ closers:
   (This will be the potential closer closest
   to the beginning of the input -- the first one in parse order.)
 
-- Now, look back in the stack (staying above `stack_bottom` and
-  the `openers_bottom` for this delimiter type) for the
-  first matching potential opener ("matching" means same delimiter).
+- Now, look back in the stack (staying above stack_bottom and the openers_bottom for this delimiter type) for the first valid potential opener, where being "valid" requires that:
+
+  + the token is a potential opener; and
+
+  + the token has the same delimiter type as the current potential closer; and
+
+  + any of the following are true:
+
+    * the closer is not a potential opener and the opener is not a potential closer; or
+
+    * the original length of the closing delimiter run is a multiple of 3; or
+
+    * the original length of the opening delimiter run + the original length of the closing delimiter run is not a multiple of 3
```

Comment:
Excellent.  Can you wrap this so that the prevailing line length of the spec is respected?
(< 80 chars/line)

Reply:
updated to match the general line length of surrounding parts (which seems to be much less than 80 chars/line at this point) and avoid changing the first bullet more than necessary

also re-added the `code` backticks which were mistakenly removed.